packages/client/src/forms/register/fieldDefinitions/register.ts

```diff
@@ -0,0 +1,12681 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import { IForm } from '@client/forms/index'
+
+/*interface IRegisterForms {
+  birth: IForm
+  death: IForm
+}*/
```

Comment:
This is the type that I plan to eventually assign to the registerForms prop below.  I am getting errors with "validation" and "mapping" types that I cant resolve easily.  In another PR I will ask for help to resolve this.